src/elm/Questionnaires/Routing.elm

```diff
@@ -11,6 +11,7 @@ type Route
     | Detail String
     | Edit String
     | Index
+    | Migrate String
```

Comment:
I would prefer to name it `Migration` to keep it consistent with the module name.